plugins/kontent-plugin/gatsby-config.js

```diff
@@ -0,0 +1,37 @@
+require("dotenv").config({
+  path: `.env.${process.env.NODE_ENV}`,
+})
+
+module.exports = {
+  siteMetadata: {
+    siteUrl: "https://gatsbykontenthomepage.gatsbyjs.io/",
```

Comment:
I believe @DSchau alluded to it in slack, but it would be great to get this hosted up on Gatsby Cloud and make this URL a reality.

I can help get this done with y'all!